Microsoft.Toolkit.Uwp.UI.Animations/ReorderGridAnimation.cs

```diff
@@ -0,0 +1,123 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using Windows.Foundation.Metadata;
+using Windows.UI.Composition;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Hosting;
+
+namespace Microsoft.Toolkit.Uwp.UI.Animations
+{
+    /// <summary>
+    /// Provides the ability to assign a reorder animation to a GridView.
+    /// </summary>
+    public class ReorderGridAnimation
+    {
+        private static readonly DependencyProperty ReorderAnimationProperty =
+            DependencyProperty.RegisterAttached("ReorderAnimation", typeof(bool), typeof(ReorderGridAnimation), new PropertyMetadata(null));
+
```

Comment:
Maybe it would be better to have typeof(ImplicitAnimationCollection) instead of typeof(bool) for maintainability :)